LK_proposals/LK Proposal for Collaboration.md

```diff
@@ -0,0 +1,116 @@
+## Proposal for Collaboration within our simulation structure
+
+There exists substantial flexibility in the conduction of adversarial collaboration (AC) (Mellers et al., 2001;  Cowan, 2020; Melloni et al., 2021). The below presented approach tries to incorporate suggestions from the literature, while also introducing deviances due to our use case.
+
+**First round: seperate studies:**
+
+In order to have a starting point of different perspectives, as well as to be able to investigate whether AC shows incremental improvements over preregistration, a preliminary simulation study should be pursued by both researchers on their own.
+
+<ins>Procedure
+
+1. 2 seperate preregistrations = simulation protocol Morris et al. (2019)
+    
+    —> remaining RDoF/QRPs in operationalization, hypothesis generation, method selection and interpretation
+    
+
+*Open question*: Should we Red Team on our opposites preregistration? Would mitigate Bias and would differ from AC, as one would not have to follow critique if it feels unwarrented
+
+2. 2 seperate implementations and interpretations
+
+<br/>
+
+**Second round: joint study**
+
+Starting point: Results of first round. Either: 
+
+1. contradicting findings
+2. subjective interpretation of ambiguous results
+3. clear reasoning for why results were not as hypothesized combined with strong opinion on what should be happening in the joint study
+
+In any case: clear divide in opinions, ideal grounds for adversary!
+
+<ins>Along the way:
+
+Constant diary entry after each following part, judging the collaboration quality in a guided fashion, oriented on the diary by Shah & Leeder (2016).
```

Comment:
This is idea is interesting. However I think it should be clear what purpose this would serve: 
1) accumulating evidence how our approach worked out, i.e. evaluation of the AC and the framework we propose.
2) improving / altering the collaborative process. In this case proposing it as a general technique for future simulations studies conducting ACs.

Reply:
I agree and will gladly add this